### DISCUSSION TITLE
yubikey authentication without a mobile

### DISCUSSION THREAD

**Discussion by Kevin-Gruber** [community/community#40071] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have two Yubikeys, I don't own a smartphone and do not plan on using one, and I don't want to give you my phone number for privacy reasons.

Why can't I use webauthn with my yubikeys without an app or a phone number ? If your concern is that I may lose my key, I have a backup one. Your policy prevents me to secure my account.

This issue has already been mentioned [there](https://github.com/orgs/community/discussions/23623), but there has not been an answer as to the why, and I think my case is different.

**Comment by airtower-luna** [comment]
You can use TOTP without a smartphone app, though unfortunately the GitHub UI doesn't mention it. Any TOTP implementation will do, and there are a variety of GUI and CLI ones. :wink:

**Reply by Kevin-Gruber** [reply]
Amazing.

Now I have the OTP seeds, MFA recovery keys, and my password store in the same password manager, because I don't have anything else I can use. This is stupid. Probably even less secure than it was before.

Nethertheless, thanks @airtower-luna . At least now I can benefit from the convenience of my Yubikeys.

**Reply by Kevin-Gruber** [reply]
Maybe in the future I could try to encrypt my MFA credentials with my Yubikeys. I will look into that.
